hermetic_build/library_generation/setup.py

```diff
@@ -24,6 +24,8 @@
             "owlbot/src/poms/*.py",
             "owlbot/templates/clirr/*.j2",
             "owlbot/templates/poms/*.j2",
+            "owlbot/templates/java_library/.github/**/*",
+            "owlbot/templates/java_library/.kokoro/**/*",
             "owlbot/templates/java_library/**/*",
```

Comment:
Do we need this line?

Reply:
yes, there are files outside `.github` and `.kokoro` that this line would include.